### PR TITLE
base_converter: Use Python-style base prefixes to auto-detect source base 

### PR DESCRIPTION
**Pull request by Keating950**
Hello,

This PR streamlines the interface for the base_converter plugin a bit by automatically detecting the source base of the input by using Python's prefix rules: `0b` means a binary literal, a leading zero specifies octal, and `0x` is (of course) hexadecimal. This is a little more convenient than having to specify the input base manually, and doesn't reduce functionality given that those three bases were the only valid options to begin with. Let me know what you think.

Thanks!

### CODE REVIEW

File: base_converter/__init__.py [hunk 1]
```diff
@@ -3,26 +3,39 @@
...
 import numpy as np
+from collections import defaultdict
 
-from albertv0 import *
+from albertv0 import Item, ClipAction
 
 __iid__ = "PythonInterface/v0.1"
 __prettyname__ = "Base Converter"
-__version__ = "1.1"
+__version__ = "2.0"
```
Comment (ManuelSchneid3r) [inline base_converter/__init__.py:19]: This is a bit messy. actually all the versions should be 0.* but well as they are they are 1.* . please use it. within each version they should be backward compatible.

**Review by ManuelSchneid3r — CHANGES_REQUESTED** [review]

File: base_converter/__init__.py (hunk 1 above)
Comment (Keating950) [reply base_converter/__init__.py:19]: Sure, I can change it to 1.2. My rationale for 2.0 was that the syntax change is not backwards compatible. Should I change it so both the old style and the one I'm suggesting work?

Comment (ManuelSchneid3r) [reply base_converter/__init__.py:19]: Thats fine, but the major version is intented to match the interface of the core. Users need no backward compatibiity for the human-machine interface :D

Comment (Keating950) [reply base_converter/__init__.py]: Sounds reasonable enough! I've reverted it to 1.1.

**Comment by ManuelSchneid3r** [comment]
trace, are there are further debugging steps you c

**Comment by ManuelSchneid3r** [comment]
674fcffb76451b3025e7994b568b55e1524b1b44

**Pull request closed without merging**